contracts/reflect/src/contract.rs

```diff
@@ -286,6 +286,6 @@ mod tests {
         )
         .unwrap();
         let value: CustomResponse = from_binary(&res).unwrap();
-        assert_eq!("DEMO ONE", value.msg.as_str());
```

Comment:
This works? I thought it didn't, so I made these changes.
But I guess that was HumanAddr and &str

Anyway, always nice to have simplificiations

Reply:
Yes, equality is defined between &str and String, and `assert_eq!` uses that